### DISCUSSION TITLE
Allow middle mouse button to open Read Post links

### DISCUSSION THREAD

**Discussion by closetgeekshow** [dailydotdev/daily#834] [Feature Requests]
This is feature request for the Chrome Extension, I'd like it if middle mouse button opened Read Post link on My Feed in new tabs without giving them focus. 

When I'm in the sort of browsing mode I'm in when on the My Feed page, I like to middle-click a bunch of things at once and then go read them. I do this every time on the My Feed page then get confused that nothing open. 

It works from the overlay that appears when you click the title, but not from the My Feed list. The behaviour I expect is on the web version but that version doesn't have all my custom shortcuts.

**Comment by rebelchris** [comment]
Hi @closetgeekshow it should work if you middle click the "Read post" hover button.
Is that not the case for you?

The one circled in orange here:
![Screenshot 2023-03-09 at 08 39 51](https://user-images.githubusercontent.com/554874/223941057-e1ebf7ff-b0ec-428a-a5ee-a91a8904b78c.png)

**Reply by closetgeekshow** [reply]
it works in the card layout but not in the list layout it seems 

![image](https://user-images.githubusercontent.com/991289/224119019-1006c305-bd86-406d-86dd-8a46a30eac59.png)

**Comment by rebelchris** [comment]
Hi @closetgeekshow if you are on the [webapp](https://app.daily.dev/) or extension version 3.24.3 it should be fixed.
Can you confirm?

**Reply by closetgeekshow** [reply]
confirmed, thank you so much
